tools/tojson.sh

```diff
@@ -28,9 +34,15 @@ then
 fi
 
 # Loop through all yaml files, convert to json
```

Comment:
Can you update the comment here? Since it's got the option to do all or just a single specified file.

Reply:
Thanks! Done